vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    include: ['**/*.vitest.(js|ts|tsx)'],
```

Comment:
As above, I would prefer to stick with the `*.test.ts` convention, and scope to codemod package to start.